index.html

```diff
@@ -97,11 +97,11 @@ <h3 class="work-title">Profesional Art Printing Data</h3>
           <li>CSS</li>
           <li>Ruby</li>
         </ul>
+        <button type="button" class="see-project-card" id="button-1">
+          See Project
+        </button>
       </div>
-      <button type="button" class="see-project-card">
-        See Project
-      </button>
-      <div class="card-content">
+      <div class="card-content" id="card-content-2">
```

Comment:
- [ ] The `see project` button should only be display on a project card on mouse hover and shouldn't be visible on the other on desktop version unlike the mobile version.

- [ ] The content of the card should be horizontally left and vertically bottom aligned as in the Figma design.
#### Design
![project-see-project-button-desk-figma](https://user-images.githubusercontent.com/44057863/167276121-c5629dc6-4a04-47d3-b449-704b87600592.png)
#### Impl
![project-see-project-button-desk-impl](https://user-images.githubusercontent.com/44057863/167276124-f7f27944-6969-42ca-b2bf-63f119ecf327.png)